src/components/GettingHelp/GettingHelpStage.vue

```diff
@@ -1,69 +1,177 @@
 <template>
-  <bf-page>
-
-    <bf-stage
-      slot="stage"
-      v-loading="isLoading"
-      element-loading-background="transparent"
-    >
-
-      <div>
-        Office Hours:
-      </div>
-
-      <div>
-        Documentation
-      </div>
-
-      <div>
-        Developers
+  <bf-stage
+    slot="stage"
+    element-loading-background="transparent"
+  >
+
+    <template #actions>
+      <a href="https://docs.pennsieve.io" target="_blank" class="mr-16"><bf-button>Visit Documentation Hub</bf-button></a>
+    </template>
+    <div>
+      <div class="highlight-image">
+        <img
+          src="/src/assets/images/illustrations/illo-dr_azumi_1.svg"
+          class="highlight"
+          alt="azumi"
+        />
       </div>
+      <div
+        class='developer-docs'
+        v-html="docSummary"
+      />
 
+    </div>
 
-    </bf-stage>
-  </bf-page>
+  </bf-stage>
 </template>
 
 <script>
+import { mapGetters } from 'vuex'
+
+import BfButton from '../shared/bf-button/BfButton.vue'
+import PaginationPageMenu from '../shared/PaginationPageMenu/PaginationPageMenu.vue'
+import ReadmeDocs from "../../mixins/readme-docs";
 
 export default {
-  name: 'GettingHelp',
+  name: 'DeveloperTools',
 
   components: {
-
+    BfButton,
+    PaginationPageMenu,
   },
 
   mixins: [
+    ReadmeDocs
   ],
 
   props: {
-
   },
 
   data() {
     return {
-      isLoading: false,
+      summary: {},
     }
   },
 
-  mounted () {
+  mounted() {
+    this.getReadmeDocument('getting-help')
   },
 
   computed: {
-  },
+    ...mapGetters([
+      'activeOrganization'
+    ]),
+    docSummary: function() {
```

Comment:
For future, a more condensed version is:

```
return this.summary ? this.summary.body_html : ''
```

it's effectively the same thing, so no change request, but the one liner reads pretty easy and takes up less space.